pkg/distributor/distributor.go

```diff
@@ -52,17 +52,19 @@ import (
 )
 
 const (
-	maxIngestionRateFlag        = "distributor.instance-limits.max-ingestion-rate"
-	maxInflightPushRequestsFlag = "distributor.instance-limits.max-inflight-push-requests"
+	maxIngestionRateFlag                 = "distributor.instance-limits.max-ingestion-rate"
+	maxInflightPushRequestsFlag          = "distributor.instance-limits.max-inflight-push-requests"
+	maxInflightPushRequestsTotalSizeFlag = "distributor.instance-limits.max-inflight-push-requests-total-size"
```

Comment:
A couple of nits:
1. I would remove "total" since it's already a "instance-limit".
2. I would add "-bytes" to clarify the unit (we applied the same pattern in other configs too)

_Same comment applies to YAML config option._

```suggestion
	maxInflightPushRequestsTotalSizeFlag = "distributor.instance-limits.max-inflight-push-requests-bytes"
```

Reply:
done in 5c99e7f007dc290726654a76e415e35bdba9088b